CautionaryAlertsApi/V1/Controllers/CautionaryAlertsApiController.cs

```diff
@@ -167,5 +170,22 @@ public async Task<IActionResult> CreateNewCautionaryAlert([FromBody] CreateCauti
                 return BadRequest("Cautionary alert cannot be created");
             }
         }
+
+        /// <summary>
+        /// Update alert to be inactive
+        /// </summary>
+        /// <response code="204">Alert updated to inactive</response>
+        [ProducesResponseType(typeof(CautionaryAlertsPropertyResponse), StatusCodes.Status200OK)]
+        [HttpPatch]
+        [Route("persons/{personId}/alerts/{alertId}")]
+        public async Task<IActionResult> EndCautionaryAlert([FromBody] EndCautionaryAlert cautionaryAlert)
+        {
+            var token = _tokenFactory.Create(_contextWrapper.GetContextRequestHeaders(HttpContext));
+
+            var result = await _endCautionaryAlertUseCase.ExecuteAsync(cautionaryAlert, token).ConfigureAwait(false);
```

Comment:
The URL implies that this is a **general** purpose PATCH endpoint, yet the use case used is `_endCautionaryAlertUseCase`.

To do: figure out whether there's anything "end cautionary alert" specific within an endpoint. If there's then that may need to be a separate endpoint, or a different URL that does not imply general purpose PATCH.